modules/sd/module.json

```diff
@@ -20,7 +20,7 @@
 			"twitter": "@ForgeVTT"
 		}
 	],
-	"url": "https://www.dndbeyond.com/sources/SD",
+	"url": "https://www.dndbeyond.com/sources/dnd/sd",
```

Comment:
This URL 404's, and I can't find anything that seems to correspond in the sources listed on the site. 🤔